packages/dev/core/src/Engines/Native/nativeInterfaces.ts

```diff
@@ -68,6 +68,7 @@ export interface INativeEngine {
     loadCubeTextureWithMips(texture: NativeTexture, data: Array<Array<ArrayBufferView>>, invertY: boolean, srgb: boolean, onSuccess: () => void, onError: () => void): void;
     getTextureWidth(texture: NativeTexture): number;
     getTextureHeight(texture: NativeTexture): number;
+    copyTexture(desination: NativeTexture, source: NativeTexture): void;

```

Comment:
nit: just so that we know it's no longer always there
```suggestion
    copyTexture?(desination: NativeTexture, source: NativeTexture): void;
```